UI/src/view/LinearAssetLayer.js

```diff
@@ -171,16 +172,29 @@ window.LinearAssetLayer = function(params) {
   map.addLayer(indicatorLayer);
   indicatorLayer.setVisible(false);
 
+  var trafficSignReadOnlyLayer = new TrafficSignReadOnlyLayer({
+      layerName: layerName,
+      style: new PointAssetStyle('trafficSigns'),
+      collection: new TrafficSignsCollection(backend, 'trafficSigns', true),
+      assetLabel: new TrafficSignLabel(),
+      assetGrouping: new AssetGrouping(9),
+      map: map
+    });
+
   var linearAssetCutter = new LinearAssetCutter(me.eventListener, vectorLayer, collection);
 
   var onSelect = function(evt) {
     if(evt.selected.length !== 0) {
       var feature = evt.selected[0];
       var properties = feature.getProperties();
       verifyClickEvent(properties, evt);
+      unHighLightReadOnlyLayer();
```

Comment:
Isn't this `unHighLightReadOnlyLayer` being already called in `verifyClickEvent ` -> `highlightMultipleLinearAssetFeatures`